### PR TITLE
hypershift: auzre install test

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -770,6 +770,17 @@ tests:
...
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi-ovn
+- as: azure-ipi-ovn-hypershift-install-p1
+  cron: 0 1 * * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;HyperShiftINSTALL&
+      TEST_SCENRAIOS: Hypershift
```
Comment (kuiwang02) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:780]: there is typo for TEST_SCENRAIOS. now it is changed to TEST_SCENARIOS

Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml]: done

**Comment by heliubj18** [comment]
I'm OK for this PR. @liangxia @kuiwang02 Please help to continue to review it. Thanks !

**Comment by kuiwang02** [comment]
I am OK.

**Pull request merged**